src/win/winapi.c

```diff
@@ -144,4 +148,10 @@ void uv__winapi_init(void) {
         ws2_32_module,
         "GetHostNameW");
   }
+
+  api_win_core_file_module = GetModuleHandleA("api-ms-win-core-file-l2-1-4.dll");
```

Comment:
TIL that this is the correct way to locate new functionality and API changes going forward (in Win10+) https://learn.microsoft.com/en-us/windows/win32/apiindex/windows-apisets#api-set-contract-names